src/main/scala/zio/redis/Input.scala

```diff
@@ -52,5 +61,19 @@ object Input {
     _6: Input[F]
   ) extends Input[(A, B, C, D, E, F)]
 
+  final case class Tuple11[-A, -B, -C, -D, -E, -F, -G, -H, -I, -J, -K](
```

Comment:
Make sure to add syntax for this to Command's companion. Also, please delete `Tuple6` as it doesn't seem to be used.

Reply:
Done